full-service/src/db/txo.rs

```diff
@@ -364,12 +408,12 @@ impl TxoModel for Txo {
             Ok(())
         })?;
 
-        Ok((
-            outlay_receiver,
-            txo_id.to_string(),
-            total_output_value as i64,
-            transaction_txo_type.to_string(),
-        ))
+        Ok(ProcessedTxProposalOutput {
+            recipient: outlay_receiver,
+            txo_id: txo_id.to_string(),
+            value: log_value as i64,
```

Comment:
Previously, this always returned the value of the overall transaction, but if we are calling this method (`create_minted`) on a change value, then the correct value to return is the value of the change.